src/server/pkg/obj/obj.go

```diff
@@ -132,7 +132,7 @@ type AmazonAdvancedConfiguration struct {
 	// uploader, and not the owner of the bucket. Using the default ensures that
 	// the owner of the bucket can access the objects as well.
 	UploadACL      string `env:"UPLOAD_ACL, default=bucket-owner-full-control"`
-	Reverse        bool   `env:"REVERSE, default=true"`
+	Reverse        bool   `env:"REVERSE, default=false"`
```

Comment:
I flipped this from 'true' to 'false' because this default has been 'false' in practice due to the silent discarding of the default value.